src/results/chp.jl

```diff
@@ -31,21 +31,27 @@
 `CHP` results keys:
 - `size_kw` Power capacity size of the CHP system [kW]
 - `size_supplemental_firing_kw` Power capacity of CHP supplementary firing system [kW]
-- `year_one_fuel_used_mmbtu` Fuel consumed in year one [MMBtu]
-- `year_one_electric_energy_produced_kwh` Electric energy produced in year one [kWh]
-- `year_one_thermal_energy_produced_mmbtu` Thermal energy produced in year one [MMBtu]
-- `year_one_electric_production_series_kw` Electric power production time-series array [kW]
-- `year_one_to_grid_series_kw` Electric power exported time-series array [kW]
-- `year_one_to_battery_series_kw` Electric power to charge the battery storage time-series array [kW]
-- `year_one_to_load_series_kw` Electric power to serve the electric load time-series array [kW]
-- `year_one_thermal_to_tes_series_mmbtu_per_hour` Thermal power to TES time-series array [MMBtu/hr]
-- `year_one_thermal_to_waste_series_mmbtu_per_hour` Thermal power wasted/unused/vented time-series array [MMBtu/hr]
-- `year_one_thermal_to_load_series_mmbtu_per_hour` Thermal power to serve the heating load time-series array [MMBtu/hr]
-- `year_one_thermal_to_steamturbine_series_mmbtu_per_hour` Thermal (steam) power to steam turbine time-series array [MMBtu/hr]
-- `year_one_chp_fuel_cost_before_tax` Cost of fuel consumed by the CHP system in year one [\$]
+- `annual_fuel_consumption_mmbtu` Fuel consumed in year one [MMBtu]
+- `annual_electric_production_kwh` Electric energy produced in year one [kWh]
+- `annual_thermal_production_mmbtu` Thermal energy produced in year one (not including waste heat) [MMBtu]
```

Comment:
Did you mean "not including curtailed thermal/heat"? If you are trying to differentiate the fuel energy that is converted to heat but is NOT recoverable (e.g. heat losses to the environment), we don't need to do that. That is implied that there is fuel energy that will be lost, thereby making the CHP overall (electric + heat thermal recovered) less than 100%.

Reply:
Answering my own question after looking at the results code a little more: so we just want to say "not including curtailed thermal", now that we switched "_to_waste" to "curtailed". I realize we are still calling the decision variable "waste" while we're changing the output name to "curtailed".

Reply:
changed in https://github.com/NREL/REopt.jl/pull/160/commits/a4fc7a09d2b5d91d8be9c8eae4bdf6ca7638d1ff